tests/Rule/LatteTemplatesRule/PresenterWithoutModule/Fixtures/VariablesPresenter.php

```diff
@@ -48,4 +48,9 @@ public function actionDirectRender(): void
         $this->template->render(__DIR__ . '/templates/Variables/direct.latte', ['fromRender' => 'b']);
         $this->terminate();
     }
+
+    public function actionDynamicInclude(): void
+    {
```

Comment:
Maybe we should add some variable which is successfully passed to included template

Reply:
Ok